include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -64,37 +64,23 @@
 #    define _DPSTD_DEPRECATED_MSG(msg)
 #endif
 
-// Check the user-defined macro for parallel policies
-#if defined(ONEDPL_USE_TBB_BACKEND)
-#    undef _ONEDPL_USE_PAR_POLICIES
-#    define _ONEDPL_USE_PAR_POLICIES ONEDPL_USE_TBB_BACKEND
-// Check the internal macro for parallel policies
-#elif !defined(_ONEDPL_USE_PAR_POLICIES)
-#    define _ONEDPL_USE_PAR_POLICIES 1
-#endif
+#define _ONEDPL_PAR_BACKEND_SERIAL 1
```

Comment:
Instead of predefining and undefining this macro, why not define it later, e.g. as a negation of `_ONEDPL_USE_PAR_POLICIES`?

Reply:
I thought about that. I don't want to use `_ONEDPL_USE_PAR_POLICIES` because the only reason I keep it for now is the code in tests.

What I write instead is something like:
```cpp
#if !_ONEDPL_PAR_BACKEND_TBB && ! _ONEDPL_PAR_BACKEND_OPENMP
#define _ONEDPL_PAR_BACKEND_SERIAL 1
#endif
```
after checking `ONEDPL_USE_TBB_BACKEND` and `ONEDPL_USE_OPENMP_BACKEND` macros